### ISSUE TITLE
Remove twisted.protocols.sip.DigestAuthorizer/BasicAuthorizer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8445]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8445|
|Type|enhancement|
|Created|2016-06-11 02:36:28Z|
|Branch|https://github.com/twisted/twisted/tree/remove-sip-digestauth-8445|

It's been deprecated for seven years and doesn't do random numbers very well or is just bad. Lets remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__8445 8445
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__remove_sip_digestauth_8445 remove-sip-digestauth-8445
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465612588731918 1465612588731918
changetime__1465630610947207 1465630610947207
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Yeah, I'm happy with this. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

go, go, go!
All good. Please merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 63f143b65779f5b8b2ed1b6cbe9350af1b37c7d5
```
#!CommitTicketReference repository="" revision="63f143b65779f5b8b2ed1b6cbe9350af1b37c7d5"
Merge remove-sip-digestauth-8445: Remove twisted.protocols.sip.DigestAuthorizer and BasicAuthorizer

Author: hawkowl
Reviewer: adiroiban
Fixes: #8445
```
